dao/src/test/java/se/fortnox/reactivewizard/db/DaoTransactionsTest.java

```diff
@@ -33,21 +34,14 @@
     private ConnectionProvider connectionProvider = db.getConnectionProvider();
     private DbProxy            dbProxy            = new DbProxy(new DatabaseConfig(), connectionProvider);
     private TestDao            dao                = dbProxy.create(TestDao.class);
-    private DaoTransactions    daoTransactions    = new DaoTransactionsImpl();
+    private DaoTransactions    daoTransactions    = new DaoTransactionsImpl(connectionProvider, dbProxy);
 
     @Test
     public void shouldRunTwoQueriesInOneTransaction() throws SQLException {
         Observable<String> find1 = dao.find();
         Observable<String> find2 = dao.find();
 
-        daoTransactions.createTransaction(find1, find2);
-
-        find1.subscribe();
-
-        db.verifyConnectionsUsed(0);
```

Comment:
You can probably keep this, to prove that calling executeTransaction does not result in connections being used, but that the `.toBlocking().singleOrDefault(null);` below will.